server/schema/clams.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE clam IF NOT EXISTS{
+    clamId INTEGER PRIMARY KEY,
```

Comment:
in a sql table we should use `underscore_case` for fields instead of `camelCase`

`keysToCamel` is the util function on the express backend that will convert these fields to `camelCase` for us when we return stuff from queries

please update the db to reflect the new schema too afterwards (hint: `UPDATE TABLE` statements) + any corresponding routes that added/changed/referenced `rakerId`